src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -130,6 +130,11 @@ define(function (require, exports, module) {
         case "BRAMBLE_DISABLE_WORD_WRAP":
             PreferencesManager.set("wordWrap", false);
             break;
+        case "BRAMBLE_CONFIGURE_AUTO_CLOSE_TAGS":
+            if(typeof args[0] === "object") {
```

Comment:
can we not simply make `args` *be* the options object? It seems unnecessary to unpack this if we're effectively simply passing the data through.